.github/ISSUE_TEMPLATE/4_figma_bug_report.yaml

```diff
@@ -4,4 +4,7 @@ labels: bug,figma,design
 body:
   - type: markdown
     attributes:
-      value: TBD
+      value: Use this template to report a bug in our [Figma libraries](TBD).
+  - type: textarea
+    attributes:
+      label: Describe the feature
```

Comment:
I think this should be "bug" instead of "feature" because it's the bug report template.